mod/ckeditor/views/default/js/elgg/ckeditor.php

```diff
@@ -1,13 +1,25 @@
+// <script>
+
 define(function(require) {
 	var elgg = require('elgg');
 	var $ = require('jquery'); require('jquery.ckeditor');
 	var CKEDITOR = require('ckeditor');
 
 	CKEDITOR.plugins.addExternal('blockimagepaste', elgg.get_site_url() + 'mod/ckeditor/views/default/js/elgg/ckeditor/blockimagepaste.js', '');
-	
+
 	var elggCKEditor = {
 
 		/**
+		 * CKEditor configuration
+		 */
+		config: (function () {
+
+			<?php elgg_view('js/elgg/ckeditor/config') ?>
```

Comment:
I'd strongly prefer not mixing php and js -- why can't we just do a `require('elgg/ckeditor/config')`?